### PR TITLE
Us138055 - add overridden confirm dialog

### PR DESCRIPTION
**Pull request by JamieTran**
[Us138055 ](https://rally1.rallydev.com/#/42960320374d/custom/236803223728?detail=%2Fuserstory%2F631594745891)

Add warning dialog for an overridden grade when saving, updating, and publishing an evaluation.

### CODE REVIEW

File: components/consistent-evaluation-page.js [hunk 1]
```diff
@@ -637,6 +644,21 @@ export default class ConsistentEvaluationPage extends SkeletonMixin(LocalizeCons
 		);
 	}
 
+	async _displayOverridenGradeWarning() {
```
Comment (JamieTran) [inline components/consistent-evaluation-page.js:647]: This function displays the dialog and waits for the dialog confirmation response.

**Comment by amauryperez** [comment]
Looks good. I was also thinking that since this is only for Quizzing if we can put into a separate file or something, it will help limit the size of `consistent-evaluation-page.js` and keep things that don't apply to all activities separated?

**Review by maddiegabriel — APPROVED** [review]
looks good to me!

**Pull request merged**